policy/modules/contrib/boothd.fc

```diff
@@ -0,0 +1,14 @@
+/etc/booth(/.*)?				gen_context(system_u:object_r:boothd_etc_t,s0)
+
+/usr/sbin/boothd			--	gen_context(system_u:object_r:boothd_exec_t,s0)
+/usr/sbin/booth-keygen			--	gen_context(system_u:object_r:boothd_exec_t,s0)
+
+/usr/lib/systemd/system/booth@\.service			--	gen_context(system_u:object_r:boothd_unit_file_t,s0)
+/usr/lib/systemd/system/booth-arbitrator\.service	--      gen_context(system_u:object_r:boothd_unit_file_t,s0)
+
+/var/run/booth(/.*)?				gen_context(system_u:object_r:boothd_var_run_t,s0)
+/var/run/booth/booth\.pid		--	gen_context(system_u:object_r:boothd_var_run_t,s0)
+/var/run/booth/localhost\.pid		--	gen_context(system_u:object_r:boothd_var_run_t,s0)
```

Comment:
Are explicit labels for the files needed somewhere given there is a regexp pattern already? Or was it meant to have it defined only for plain files?

Reply:
The explicit labels for the files weren't necessary, so I removed them